plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/hms/HiveMetastoreTableOperations.java

```diff
@@ -97,7 +99,15 @@ protected void commitToExistingTable(TableMetadata base, TableMetadata metadata)
             }
         }
         finally {
-            thriftMetastore.releaseTableLock(lockId);
+            try {
+                thriftMetastore.releaseTableLock(lockId);
+            }
+            catch (Exception e) {
```

Comment:
Can we narrow this down?

for example why not `RuntimeException`?
can it be even more specific?

Reply:
`ThriftHiveMetastore.Client` which is used by [ThriftHiveMetastoreClient](https://github.com/trinodb/trino/blob/113f22d8d770f16d0edca2eeeb05be036101fc80/plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftHiveMetastoreClient.java#L642) throws `TException` that extends  `Exception`.
The specific exception like `NoSuchLockException` can be thrown by the metastore that extends `TException`

So, maybe we can catch `TException`?

Reply:
`thriftMetastore.releaseTableLock` throws no checked exceptions, so `RuntimeException` would suffice